sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplatePartitionIT.java

```diff
@@ -37,32 +37,18 @@
 import org.springframework.data.annotation.Persistent;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
 import org.springframework.data.repository.query.parser.Part;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
-
-import static com.azure.spring.data.cosmos.common.TestConstants.ADDRESSES;
```

Comment:
Not sure why it says I made this change, I have merged i the latest. Perhaps a merge issue?

Reply:
It probably is caused by your intellij config, instead of using single class import, it combines into import *

You can change in Preferences -> CodeStyle -> Java -> Imports